app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -62,6 +62,10 @@ class LuciBuildService {
   static const int kDefaultPriority = 30;
   static const int kRerunPriority = 29;
 
+  /// Github labels have a max length of 100, so conserve chars here.
```

Comment:
Nit: doc that these are used for packages repo only, and nice to add a link pointing to the pacakges recipes.